pdfmap/pdfmaze.py

```diff
@@ -21,50 +21,111 @@ class PDFMaze:
+                    confidence=confidence
+                )
+
+                textboxes.append(textbox)
+
+            # if it's a textbox or a container, also recurse
+            if isinstance(
+                obj,
+                (LTTextBoxHorizontal, LTFigure)
+            ):
+                other_textboxes = self._parse_objs(obj._objs, confidence=confidence)
+                textboxes.extend(other_textboxes)
 
+        else:
+            x1, y1, x2, y2, text = -1, -1, -1, -1, ''
+            if isinstance(obj, LTText):
+                for line in obj:
+                    for char in line:
+                        # If the char is a line-break or other symbol chosen by 
+                        #  the user, the word is complete
+                        if isinstance(char, LTAnno) or char.get_text() in key_split_chars:
+                            if x1 != -1:
+                                # If the char is a line-break, get the coordinates
+                                #  of the previous char
+                                if not isinstance(char, LTAnno):
+                                    x2, y2, = char.bbox[2], char.bbox[3]
+                                else:
+                                    x2, y2, = previous_char.bbox[2], previous_char.bbox[3]
+                                    
+                                textbox = TextBox(
+                                    x1=x1,
+                                    x2=x2,
+                                    y1=y1,
+                                    y2=y2,
+                                    text=text,
+                                    confidence=confidence
+                                )
+
+                                textboxes.append(textbox)
+
+                            x1, y1, x2, y2, text = -1, -1, -1, -1, ''
+                        elif isinstance(char, LTChar):
+                            text += char.get_text()
+                            if x1 == -1:
+                                x1, y1, = char.bbox[0], char.bbox[1]
+                        previous_char = char
+
+            # If the last symbol in the PDF was neither other symbol chosen
+            #  by the user nor a line-break, add the last word to the word_map
+            if x1 != -1:
+                x2, y2, = char.bbox[2], char.bbox[3]
+                textbox = TextBox(
+                    x1=x1,
+                    x2=x2,
+                    y1=y1,
+                    y2=y2,
+                    text=text,
+                    confidence=confidence
+                )
+
+                textboxes.append(textbox)
+        
         return textboxes
 
+
     def parse_pdf(
             self,
             source: Union[str, PathLike, bytes],
             origin: Origin = Origin.TOP_LEFT,
-            confidence: Optional[Real] = None
+            confidence: Optional[Real] = None,
+            split_words: bool = False,
+            key_split_chars: list = [' ']
```

Comment:
Same as previous comment:
```suggestion
            key_split_chars: Iterable = (' ',)
```